app/models/basic_container_image.rb

```diff
@@ -0,0 +1,4 @@
+class BasicContainerImage < ContainerImage
```

Comment:
It will make it easy to address all the container images that don't have an entity in Openshift/OtherContainerProvider managing them. It will hopefully be handy with the UI.